cmd/stack.go

```diff
@@ -19,11 +19,13 @@ import (
 	"fmt"
 	"sort"
 
-	"github.com/dustin/go-humanize"
+	humanize "github.com/dustin/go-humanize"
```

Comment:
for my own edification, why did this need to change?

Reply:
I think this is the fault of `GoImports`, which is a `vim` plugin I use to automagically add imports for APIs I've used. `GoImports` does this when a package's name does not match the terminal element of its import path.